Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -22,13 +22,21 @@ def install
     (buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve").install Dir["{*,.git,.gitignore}"]
 
     ENV["GOPATH"] = "#{buildpath}/gopath"
-
-    cd buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve" do
+    build_root = buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve"
+    cd build_root do
       if build.head?
         git_hash = `git rev-parse --short HEAD --quiet`.chomp
-        git_hash = " HEAD-#{git_hash}"
+        git_hash = "HEAD-#{git_hash}"
+        ENV["CGO_LDFLAGS"] = "#{build_root}/vendor/build/lib9p/lib9p.a -L#{build_root}/vendor/lib9p"
+        ENV["CGO_CFLAGS"] = "-I#{build_root}/vendor/lib9p"
+        mkdir "vendor/build"
```

Comment:
I can do that. I'm a novice (read: complete beginner) at both go and ruby, and a quick search for a ruby-equivalent of ```mkdir -p``` made me think it didn't exist. Upstream has changed HEAD to partially account for this ticket, so let me sync the formula up.